testdata/filters/rbac.yaml.tmpl

```diff
@@ -6,7 +6,7 @@
       rules:
         action: DENY
         policies:
-          "test":
+          ns[foo]-policy[httpbin-deny]-rule[0]:
```

Comment:
Do we need an example for network RBAC?

Reply:
Should be similar to this one with response to extracting stuff? I can add if needed..

Reply:
Could be done in a follow-up. Just want to make sure we have coverage for network filter as well.

Reply:
Sure, will add one